homeassistant/components/min_max/sensor.py

```diff
@@ -110,6 +113,26 @@ def calc_mean(sensor_values, round_digits):
     return round(sensor_value_sum / count, round_digits)
 
 
+def calc_median(sensor_values, round_digits):
+    """Calculate median value, honoring unknown states."""
+    count = 0
+    sensor_value_list = []
+    for _, sensor_value in sensor_values:
+        if sensor_value != STATE_UNKNOWN:
+            count += 1
```

Comment:
Use `len(sensor_value_list)` instead of keeping track of count.

Reply:
Agreed, changed both mean and median calculations to list operations instead.